src/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <div class="container">
+    <h1 class="my-5">Leaderboard</h1>
+    <main class="row">
+      <section id="recent-scores" class="col-md-6 px-3">
+        <div id="recent-scores-head" class="d-flex justify-content-around">
+          <h2>Recent Scores</h2>
+          <button class="my-auto btn btn-light border border-dark">Refresh</button>
+        </div>
+        <div id="recent-scores-body" class="px-md-3 mt-3 ">
+          <table class="table table-light table-sm table-striped table-hover border border-dark">
+            <tbody>
+              <tr>
+                <td>Name</td>
+                <td>80</td>
+              </tr>
+              <tr>
+                <td>Name</td>
+                <td>40</td>
+              </tr>
+              <tr>
+                <td>Name</td>
+                <td>100</td>
+              </tr>
+              <tr>
+                <td>Name</td>
+                <td>55</td>
+              </tr>
+              <tr>
+                <td>Name</td>
+                <td>200</td>
+              </tr>
+            </tbody>
+          </table>
```

Comment:
- [ ] OPTIONAL: This table could have been better if you constructed it using JS DOM. It represents dynamic data when you progress to the next milestone in which hitting API is required to fetch data. Even though it is not mentioned explicitly, the project requirements of this milestone have an ES6 JS module/s. :pray: